content/en/tracing/trace_ingestion/mechanisms.md

```diff
@@ -207,3 +207,7 @@ Some additional ingestion reasons are attributed to spans that are generated by
 [1]: /tracing/setup_overview/setup/
 [2]: /tracing/visualization/#trace-root-span
 [3]: /tracing/trace_ingestion/control_page
+[4]: /synthetics/apm/
+[5]: /real_user_monitoring/connect_rum_and_traces/
+[6]: /serverless/distributed_tracing/
```

Comment:
```suggestion
[6]: /serverless/libraries_integrations/extension/
```

Reply:
Is this link intended to point to the Lambda extension, as the link text on line 199 would suggest? If the link is indeed meant for /serverless/distributed_tracing/, could you update the link text to "serverless applications" or something similar?